### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by kevkev912** [codetheweb/tuyapi#404] [Q&A]
Hi, this might be a pretty stupid question, but where do I put the code respectively how do I use it?
Sorry if it is too dumb

**Comment by codetheweb** [comment]
It depends on what you're trying to do. To run the example in the README, copy/paste it into something like `tuya.js`, install the package with `npm i tuyapi`, then run the script with `node tuya.js`.

I would recommend looking elsewhere for general programming help.
